requirements-dev.txt

```diff
@@ -3,5 +3,8 @@ parsel>=1.5.2
 jmespath>=0.9.5
 itemadapter>=0.1.0
 
-pytest==5.4.1
```

Comment:
I wonder what is this file about tbh. It's not mentioned anywhere and I don't think it's needed for development.

Reply:
The "requirements-dev.txt" file contains all the necessary packages for local development, which you need to install in your virtual environment to work with the package. 

Although Pytest may already be installed in the global system,but I prefer to have it in the local env too. Also, while pre-commit installs Black, isort, and flake8 automatically, it's helpful to have them installed, in order to run them independently. That said, if you prefer to remove them, you can do so.

Reply:
Shouldn't you use tox, not pytest locally?

Reply:
I'm actually not sure why we need requirements-dev.txt :)

Reply:
PS: https://github.com/scrapy/itemloaders/blob/master/tox.ini seems outdated on the Python version front.